js/binance.js

```diff
@@ -4897,4 +4899,24 @@ module.exports = class binance extends Exchange {
     async addMargin (symbol, amount, params = {}) {
         return await this.modifyMarginHelper (symbol, amount, 1, params);
     }
+
+    async fetchBorrowRate (code, params = {}) {
+        await this.loadMarkets ();
+        const currencyId = this.currency (code)['id'];
+        const request = {
+            'asset': currencyId,
+            // 'vipLevel': this.safeInteger (params, 'vipLevel'),
+        };
+        const response = await this.sapiGetMarginInterestRateHistory (this.extend (request, params));
+        const rate = this.safeValue (response, 0);
+        const timestamp = this.safeNumber (rate, 'timestamp');
+        return {
+            'currency': currencyId,
```

Comment:
We will need to put a unified code instead of an exchange-specific currency id here.